app/docker/views/volumes/create/createVolumeController.js

```diff
@@ -99,7 +99,7 @@ angular.module('portainer.docker').controller('CreateVolumeController', [
       }
       driverOptions.push({ name: 'o', value: options });
 
-      var mountPoint = data.mountPoint[0] === ':' ? data.mountPoint : ':' + data.mountPoint;
+      var mountPoint = (data.mountPoint.indexOf(":") === -1 && data.mountPoint[0] === ":") ? ":" + data.mountPoint : data.mountPoint;
```

Comment:
```js
data.mountPoint.indexOf(":") === -1 && data.mountPoint[0] === ":"
```
This will always be false because `indexOf === -1` means there is no `:` so `mountPoint[0]` cannot be `:`

It should be
```js
const mountPoint = data.mountPoint.includes(":") ? data.mountPoint : `:${data.mountPoint}`;
```

* `/some/path` -> includes = false -> `:/some/path` (add `:`)
* `:/some/path` -> includes = true -> `:/some/path` (use `data.mountPoint`)
* `192.168.1.1:/some/path` -> includes = true -> `192.168.1.1:/some/path` (use `data.mountPoint`)